Description: Fix texinfo error due to wrong @subsubsection
 The decoders.texi contains a @subsubsection directly below a @section,
 which leads to the following error from texinfo:
 raising the section level of @subsubsection which is too low

Author: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Forwarded: <https://ffmpeg.org/pipermail/ffmpeg-devel/2014-March/155976.html>
Applied-Upstream: <https://ffmpeg.org/pipermail/ffmpeg-devel/2014-March/156008.html>
Last-Update: <2014-03-24>

--- ffmpeg-2.2.orig/doc/decoders.texi
+++ ffmpeg-2.2/doc/decoders.texi
@@ -62,7 +62,7 @@ AC-3 audio decoder.
 This decoder implements part of ATSC A/52:2010 and ETSI TS 102 366, as well as
 the undocumented RealAudio 3 (a.k.a. dnet).
 
-@subsubsection AC-3 Decoder Options
+@subsection AC-3 Decoder Options
 
 @table @option
 
